packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts

```diff
@@ -67,13 +67,14 @@ export class AzureCommunicationMeetingAdapter implements MeetingAdapter {
   }
 
   public joinMeeting(microphoneOn?: boolean): void {
-    throw new Error('Method not implemented.');
+    this.callAdapter.joinCall(microphoneOn);
   }
-  public async leaveMeeting(forEveryone?: boolean): Promise<void> {
-    throw new Error('Method not implemented.');
+  public async leaveMeeting(): Promise<void> {
+    await this.chatAdapter.removeParticipant(this.chatAdapter.getState().userId);
+    await this.callAdapter.leaveCall();
   }
   public startMeeting(participants: string[]): void {
-    throw new Error('Method not implemented.');
+    this.callAdapter.startCall(participants);
```

Comment:
I'm a bit confused by this API. I understand that using the `MeetingComposite`, we can join an existing Teams meeting.

But what does starting a new meeting mean? How would we get an associated chat thread?